analysis/pcmdimetrics/portrait4seasv1p1p1.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+#################################################################################
+#  RELIES UPON CDAT MODULES 
+#  PRODUCE SIMPLE PORTRAIT PLOT BASED ON RESULTS FROM PCMDI'S METRICS PACKAGE 
+#  WRITTEN BY P. GLECKLER, PCMDI/LLNL
+#  FURTHER TWEAKED BY C. DOUTRIAUX
+#  ISSUE TO BE RESOLVED WITH UV-CDAT PLANNED IMPROVEMENTS: ABILITY TO SAVE PLOTS IN MULTIPLE FORMATS  
+#################################################################################
+
+# CHECK TO VERIFY VCS MODULE IS INSTALLED
+try:
+    import vcs
+except:
+    raise RuntimeError("Sorry your python is not build with VCS support and cannot generate portrait plots")
```

Comment:
`vcs` is a python package available in PyPi (https://pypi.python.org/pypi/vcs).  The message should indicate that package is not installed, not that python was not built with VCS support.